examples/list_disk_snapshots.py

```diff
@@ -64,8 +64,12 @@ def parse_args():
     # Create mapping from snapshot parent to snapshot, used to reconstruct the
     # snapshot chain.
     snapshots = {}
-    for s in disk_snapshots_service.list(include_active=True):
+
+    # Snapshots belonging to other disks will be added here
+    other_snapshots = []
+    for s in disk_snapshots_service.list(include_active=True, include_template=True):
         if s.disk.id != disk.id:
+            other_snapshots.append(s)
             continue
         parent = s.parent.id if s.parent else None
```

Comment:
Can lines 74-82 become a helper method and then reused (lines 89-97)?

Reply:
definitely